tests/components/here_travel_time/test_sensor.py

```diff
@@ -312,141 +430,57 @@ async def test_invalid_origin_entity_state(hass, caplog, valid_response: MagicMo
     assert "test_state are not valid coordinates" in caplog.text
 
 
-async def test_route_not_found(hass, caplog):
+async def test_route_not_found(hass: HomeAssistant, caplog):
     """Test that route not found error is correctly handled."""
-    config = {
-        DOMAIN: {
-            "platform": PLATFORM,
-            "name": "test",
-            "origin_latitude": CAR_ORIGIN_LATITUDE,
-            "origin_longitude": CAR_ORIGIN_LONGITUDE,
-            "destination_latitude": CAR_DESTINATION_LATITUDE,
-            "destination_longitude": CAR_DESTINATION_LONGITUDE,
-            "api_key": API_KEY,
-        }
-    }
     with patch(
-        "homeassistant.components.here_travel_time.sensor._are_valid_client_credentials",
-        return_value=True,
+        "homeassistant.components.here_travel_time.config_flow.validate_api_key",
```

Comment:
Why are we patching a name in the config flow here? The config flow isn't used here, right?